parser/src/java/org/apache/hadoop/hive/ql/parse/IdentifiersParser.g

```diff
@@ -720,7 +720,7 @@ partitionVal
 
 dropPartitionSpec
     :
-    KW_PARTITION
+    KW_PARTITION | KW_PARTITIONS
```

Comment:
I think this should be `(KW_PARTITION|KW_PARTITIONS)`  ; because in it's current form it will
* accept `PARTITION`
* and `PARTITIONS  (...)`
* but not  `PARTITION (...)`

Reply:
As discussed above!